packages/admin/README.md

```diff
@@ -61,50 +61,50 @@ Please, refer to the implementation for more details.
 
 ## Requester commands
 
-### `derive-designated-wallet`
+### `derive-sponsor-wallet`
 
 Derives the address of the [wallet designated by an Airnode for a requester](https://github.com/api3dao/api3-docs/blob/master/request-response-protocol/designated-wallet.md).
```

Comment:
This should probably be updated as well? (At least the filename should be renamed)

Reply:
This is something I wasn't sure what to do. If I update the file name then the link will be broken until the docs are updated.

Reply:
I am not sure how the docs are versioned, but the documentation in the docs should definitely update as well.
That said, it can be done afterwards. 

@bbenligiray @wkande what is the best approach for this?

Reply:
We have https://docs.api3.org/next/ so it should be probably updated there.

I am not sure if you want to wait until it's done, but we should at least write a TODO: in comment to update it later (https://stackoverflow.com/questions/46734820/hidden-markdown-text-on-github/46734863)

Reply:
I think the main problem here is that the docs and these READMEs are in separate repos. I'd rather keep the READMEs as light as possible and have these in the docs repo. Let's talk about it at the call next week.